desktop/app.js

```diff
@@ -103,7 +103,9 @@ module.exports = function main() {
       Menu.setApplicationMenu(
         Menu.buildFromTemplate(createMenuTemplate(args), mainWindow)
       );
-      nativeTheme.themeSource = settings.theme;
+      if ('theme' in settings) {
```

Comment:
Just want to make sure you know the implications of using `in` over `hasOwnProperty`. In does not care if that property is directly on the object or inherited. There are also some other differences. This seems like a pretty good [article](https://masteringjs.io/tutorials/fundamentals/hasownproperty). (I only skimmed it)

In this case the use of `in` is fine.

Reply:
Thank you!